waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/DataFlowAction.scala

```diff
@@ -41,14 +42,20 @@ trait DataFlowAction[T, C] {
 
   def logLabel = s"$guid: $description"
 
+
```

Comment:
Could not find anywhere where it is used. Plus there is function DataFlow.executed which will update the inputs part of the next DataFlow state.